js/utils.js

```diff
@@ -1049,8 +1049,8 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     var isSortAsc = options.sortOrder === 'asc';
     var sortField = options.sortField;
     var startsWithAlphabet = /^[A-Z,a-z]/;
-    var sortType = getFieldType(records[0][sortField]);
-  
+    var sortType = getFieldType(records[0].data ? records[0].data[sortField]: records[0][sortField]);
```

Comment:
Why would the fields be nested inside a `data` object and how is that related to the issue OD-222?

Reply:
They are already nested in `data`
![image](https://user-images.githubusercontent.com/74735207/102098039-5a0b3100-3e2f-11eb-8a20-cfd2acaa76a7.png)
And we have same condition below on row 1055:
`var aValue = a.data ? a.data[sortField] : a[sortField];`